pkg/proxy/integrations/grpcparser/fake_server.go

```diff
@@ -259,27 +259,27 @@ func (srv *transcoder) ProcessContinuationFrame(ContinuationFrame *http2.Continu
 func (srv *transcoder) ProcessGenericFrame(frame http2.Frame) error {
 	//PrintFrame(frame)
 	var err error
-	switch frame.(type) {
+	switch frame := frame.(type) {
 	case *http2.PingFrame:
-		err = srv.ProcessPingFrame(frame.(*http2.PingFrame))
+		err = srv.ProcessPingFrame(frame)
```

Comment:
Why is the cast removed it will throw an error right if ProcessPingFrame is expecting http2.PingFrame

Reply:
<img width="409" alt="image" src="https://github.com/keploy/keploy/assets/60891544/6ebaac22-ae5b-4758-aa96-7f18a31b54dd">
because this would be a better approach